driver/src/contracts/stablex_contract.rs

```diff
@@ -128,8 +128,7 @@ impl StableXContract for BatchExchange {
             prices,
             token_ids_for_price,
         )
-        .gas(5_000_000.into())
-        .gas_price(20_000_000_000u64.into())
+        .gas_price(GasPrice::Scaled(3.0))
```

Comment:
Safe to assume this is only possible because of the recent ethcontract update?

Reply:
I don’t really see any reference to the word estimate here. I suppose then that current gas price means the current average

Reply:
So `GasPrice::Scaled` will scale the average gas price by a factor of 3.0. Eventually, we can integrate something like ETH Gas Station as well and just use the "fast" gas price.